shell/platform/darwin/macos/framework/Source/FlutterEngineTestUtils.mm

```diff
@@ -4,11 +4,44 @@
 // found in the LICENSE file.
 
 #import "flutter/shell/platform/darwin/macos/framework/Source/FlutterEngineTestUtils.h"
+
 #import "flutter/shell/platform/darwin/macos/framework/Source/FlutterDartProject_Internal.h"
 #import "flutter/shell/platform/darwin/macos/framework/Source/FlutterEngine_Internal.h"
 
 #include "flutter/testing/testing.h"
 
+// Fake pasteboard implementation to allow tests to work in environments without a real
+// pasteboard.
+@interface FakePasteboard : FlutterPasteboard
+@end
+
+@implementation FakePasteboard {
+  NSString* _result;
+}
+
+- (instancetype)init {
+  self = [super init];
+  if (self != nil) {
+    _result = nil;
```

Comment:
This is a no-op; ObjC class ivars are automatically initialized to whatever the type's equivalent of zero is. So the entire `init` override can be removed.